méli-mélo/2023-09-menu/campaign-menu.js

```diff
@@ -203,8 +161,13 @@
                         // Set focus on submenu
                         if (savedFocusHref.match("^#")) {
                             let gcSubmenuFocus = $(savedAnchorFocus).attr("aria-controls");
-                            let submenuId = gcSubmenuFocus.substring(9);
-                            $(".wb-menu a[href='#menu-" + submenuId + "']").focus();
+                            let submenuId = gcSubmenuFocus.substring(12);
```

Comment:
```
.substring( 12 )
```
That instruction look like to be very chirurgical. Not an issue for this PR, but it will be preferable to calculate the length to be used by the substring instead of using a fix length.